data/transit/route/bus.json

```diff
@@ -2904,10 +2904,13 @@
     },
     {
       "displayName": "EMTU",
-      "id": "emtu-2ef076",
+      "id": "emtuempresametropolitanadetransportesurbanos-2ef076",
       "locationSet": {"include": ["br"]},
       "tags": {
-        "network": "EMTU",
+        "network": "EMTU - Empresa Metropolitana de Transportes Urbanos",
```

Comment:
The effect of this is that, once we deploy this, iD validator will suggest a tag upgrade for every network=EMTU bus route in Brazil to use the long value that you have changed it to. Is this what you intended?

In my opinion there is no harm in continuing to use network=EMTU tag, but there is clear benefit in adding the network:wikidata and network:wikipedia tags.

Reply:
This was getting a bit stale, so I just added the tags in 1fe62781.. Thanks @Tubaleviao 👍